src/test/java/Assignment16Test.java

```diff
@@ -0,0 +1,19 @@
+import com.assignment.Yes1;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import com.assignment.*;
+
+public class Assignment16Test {
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    @Test
+    public void testAbstractClass() {
+        int integerToSum = 1;
+        Yes1 yes1 = new Yes1();
```

Comment:
remove this line,use new Yes1().sum(integerToSum) directly

Reply:
i cannot do that unless i declare Yes1 as static